scripts/compare-versions/src/main.rs

```diff
@@ -150,6 +151,9 @@ fn main() -> Result<()> {
             print_selected_versions(&[latest_sway_version], &[latest_fuel_core_version]);
             std::process::exit(0);
         }
+        Err(_) => {
+            bail!("Unexpected error trying to fetch channel - retrying at the next scheduled time");
```

Comment:
Should we include the error here so that a user can help us debug? E.g.

```rust
    Err(e) => {
        bail!("Unexpected error trying to fetch channel: {} - retrying at the next scheduled time");
```